oem/ibm/libpldmresponder/oem_ibm_handler.cpp

```diff
@@ -1227,7 +1346,8 @@ void pldm::responder::oem_ibm_platform::Handler::handleBootTypesAtPowerOn()
         }
         catch (const std::exception& e)
         {
-            return;
+            std::cerr << "Failed to fetch the restartCause, ERROR=" << e.what()
+                      << "\n";
```

Comment:
this is an unrelated change , please remove

Reply:
done